.drone.yml

```diff
@@ -130,6 +130,28 @@ pipeline:
       - rm $BIN/vic_ui_*.tar.gz
     when:
       status:  success
+      branch: [ master, releases/* ]
```

Comment:
I think we want releases branch below

Reply:
so it covers both tag events and changes to our releases branch

Reply:
ok so `vic-ui` should be triggered only on `master` and `vic-ui-release` on `releases/*` and `refs/tags/*` right?